src/views/TimezoneModal.vue

```diff
@@ -114,23 +133,26 @@ export default defineComponent({
     },
     preventSpecialCharacters($event: any) {
       // Searching special characters fails the API, hence, they must be omitted
-      if(/[`!@#$%^&*()_+\-=\\|,.<>?~]/.test($event.key)) $event.preventDefault();
+      if (/[`!@#$%^&*()_+\-=\\|,.<>?~]/.test($event.key)) $event.preventDefault();
     },
-    findTimeZone() { 
+    findTimeZone() {
       const queryString = this.queryString.toLowerCase();
       this.filteredTimeZones = this.timeZones.filter((timeZone: any) => {
         return timeZone.id.toLowerCase().match(queryString) || timeZone.label.toLowerCase().match(queryString);
       });
     },
     async getAvailableTimeZones() {
       const resp = await UserService.getAvailableTimeZones()
-      if(resp.status === 200 && !hasError(resp)) {
+      console.log(resp);
```

Comment:
Remove console statement from the code, as we use this console statements for debugging hence should not be pushed.